common/remote_util.cc

```diff
@@ -119,7 +119,7 @@ ProcessStartInfo RemoteUtil::BuildProcessStartInfoForSsh(
 
 ProcessStartInfo RemoteUtil::BuildProcessStartInfoForSshPortForward(
     int local_port, int remote_port, bool reverse) {
-  // (internal): Usually, one would pass in -N here, but this makes the
+  // Usually, one would pass in -N here, but this makes the
```

Comment:
Supernit: Reformat line breaks, here and in the other multi-line comments where you dropped `internal`.